### ISSUE TITLE
run on CPU?

### ISSUE THREAD

**Issue by m3m3-chan** [ChengeYang/Skeleton-Based-Human-Action-Recognition#4]
Hello, 

Thanks for sharing this great repo!
Can I run the code on CPU or is GPU a must?

thanks,

**Comment by trilokpadhi** [comment]
Yes you can run on CPU , it will get slower and you will get a low FPS , I got around 0.7 FPS
